open-source-hosts-plugin/README.md

```diff
@@ -30,6 +30,8 @@ To debug the plugin, press ctrl+alt+I (or cmd+option+I on a mac) to bring up the
 
 ### Add a host to the scene
 
+**Note:** Please disable the SSAO (Scene -> Inspector -> Rendering -> Disable SSAO 2) to fix the lighting issue until this problem gets fixed by BabylonJS Editor.
```

Comment:
A few improvements I'd recommend:

- Give the user a little more context on the issue. Recognize that they won't know what you're referring to unless you tell them.
- Don't assume the user knows what the acronym "SSAO" stands for. Refer to it more descriptively.
- Call out which version of the Babylon.js Editor is effected.

Here's some possible wording...

> There is a known issue that causes undesirable shadowing artifacts when using Host characters. This is due to a bug in Babylon.js Editor version x.x.x. Until this bug is fixed, you can avoid the issue by disabling the Screen Space Ambient Occlusion (SSAO) post-processing effect for your scene. **Scene -> Inspector -> Rendering -> Disable SSAO 2**

Reply:
Thank you for the wording suggestion!